jobs/build/ocp/Jenkinsfile

```diff
@@ -505,7 +514,8 @@ node(TARGET_NODE) {
           buildlib.write_sources_file()
           buildlib.oit """
 --working-dir ${OIT_WORKING} --group 'openshift-${BUILD_VERSION}'
-distgits:rebase --sources ${env.WORKSPACE}/sources.yml --version v${NEW_VERSION}
+--sources ${env.WORKSPACE}/sources.yml
```

Comment:
--sources was moved before the verb because is now part of runtime since we use it for both images and rpms. Or at least can. rpms don't strictly need it, but will do the right thing if it's given.